component/main.jsonnet

```diff
@@ -137,6 +137,31 @@ local syncCategoriesContainer = {
   ],
 };
 
+local tenantMappingCJ = common.CronJob('tenant-mapping', 'tenantmapping', {
+  initContainers: [
+    checkMigrationContainer,
+  ],
+  containers: [
+    {
+      name: 'mapping',
+      image: formatImage(params.images.reporting),
+      env+: dbEnv + promEnv,
+      command: [ 'sh', '-c' ],
+      args: [
+        'appuio-cloud-reporting tenantmapping --begin=$(date -d "now -3 hours" -u +"%Y-%m-%dT%H:00:00Z") --repeat-until=$(date -u -Iseconds)'
```

Comment:
What's the reasoning for doing this for the last three hours once an hour? Is this just to make sure we don't miss anything?

side-note: I see that we do the same thing for the `check-missing` cronjob, which also runs once an hour.

Reply:
Because of the varying lag of Thanos getting updated. So we don't miss anything if Thanos is not yet updated.